pkg/controllers/podgroup/pg_controller_handler.go

```diff
@@ -49,6 +55,44 @@ func (pg *pgcontroller) addPod(obj interface{}) {
 	pg.queue.Add(req)
 }
 
+func (pg *pgcontroller) deletePod(obj interface{}) {
```

Comment:
Are we going to delete `PodGroup` when one Pod deleted?

Reply:
> IMO, we should handle this case by GC, e.g. only delete PodGroup when its owner was deleted.

I agree with you,the essence of this problem is wrong to set the owner of pod,but it's a little hard to find the outest controller by pod,there are too many upstream pods that can be created,and we cannot enumerate explicit outputs.

Reply:
> > IMO, we should handle this case by GC, e.g. only delete PodGroup when its owner was deleted.
> 
> I agree with you,the essence of this problem is wrong to set the owner of pod,but it's a little hard to find the outest controller by pod,there are too many upstream pods that can be created,and we cannot enumerate explicit outputs.

We can find controlled owner reference of Pod to do that :)

Reply:
ok, let me change that